sdk/template/azure-template/inc/azure/template/dll_import_export.hpp

```diff
@@ -31,3 +37,17 @@
 #endif // AZ_STORAGE_BLOBS_BUILT_AS_DLL
```

Comment:
```suggestion
#endif // AZ_TEMPLATE_BUILT_AS_DLL
```

Reply:
Thank you! There were other places like this, I fixed them all now :)